imio/dms/mail/examples.py

```
@@ -437,8 +439,9 @@ def add_test_mails(context):
     ifld = site["incoming-mail"]
     data = DummyView(site, site.REQUEST)
     for i in range(1, 10):
-        if not "courrier%d" % i in ifld:
-            scan_date = receptionDateDefaultValue(data)
+        scan_date = receptionDateDefaultValue(data)
+        sub_folder = create_period_folder(ifld, scan_date)
+        if not "courrier%d" % i in sub_folder:
```

Ruff (0.15.21)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)